src/services/flightsDataService.js

```diff
@@ -9,9 +9,8 @@ const rootURL = "http://localhost:3030/";
 
 
 export const getFlights = async () => {
-    console.log('wooooo')
-    // const response = await axios.get(`${ rootURL }flights` );
-    return ['hello', 'world'];
+    const response = await axios.get(`${ rootURL }flights` );
```

Comment:
Pro tip, with a known API you can set up an axios instance.

You supply to it the root URL (so you don't have to keep passing it), and then any headers you want to be reused (such as auth?) in addition to reusable config items such as timeout.

Reply:
Will push this pro tip into my next PR.